src/main/java/kobe/Commands.java

```diff
@@ -0,0 +1,37 @@
+package kobe;
+
+import java.lang.System;
+
+public class Commands {
+    public static String ind = "    ";
```

Comment:
I think you want to put "ind" here as indentation. But i think it should be more expressive because it's attribute. Can be "indentation".